derive/src/pyclass.rs

```diff
@@ -523,13 +528,35 @@ pub fn impl_pystruct_sequence(attr: AttributeArgs, item: Item) -> Result<TokenSt
                 );
                 ::rustpython_vm::pyobject::PyValue::into_ref_with_type(tuple, vm, cls)
             }
+
+            fn repr(zelf: rustpython_vm::pyobject::PyRef<rustpython_vm::obj::objtuple::PyTuple>, vm: &rustpython_vm::VirtualMachine) -> ::rustpython_vm::pyobject::PyResult<String> {
+                let s = if let Some(_guard) = rustpython_vm::vm::ReprGuard::enter(zelf.as_object()) {
+                        let field_names = vec![#(stringify!(#field_names)),*];
+                        let mut fields = Vec::new();
+                        for (value, field_name) in zelf.as_slice().iter().zip(field_names.iter()) {
+                                let s = vm.to_repr(value)?;
+                                fields.push(format!("{}: {}", field_name, s));
+                        }
+
+                        if fields.len() == 1 {
+                                format!("{}({},)", #py_class_name, fields[0])
+                        } else {
+                                format!("{}({})", #py_class_name, fields.join(", "))
+                        }
+                } else {
+                        concat!(#class_name, "(...)").to_string()
```

Comment:
Without checking, I feel like CPython may use `__name__` and `__module__` here. Though it will work because these kind of builtin types doesn't allow to set `__name__` and `__module__`